overlay/files/system/oem/06_recovery.yaml

```diff
@@ -0,0 +1,30 @@
+# Default Kairos OEM configuration file
+#
+# This file is part of Kairos and will get reset during upgrades.
+#
+# Before you change this file manually,
+# consider copying this file to /usr/local/cloud-config or
+# copy the file with a prefix starting by 90, e.g. /oem/91_custom.yaml
+name: "Recovery partition boot setup"
+stages:
+   rootfs.before:
+     - if: |
+            [ -n "$(blkid -L COS_SYSTEM || true)" ] || cat /proc/cmdline | grep -q "COS_RECOVERY"
+       name: "Identify recovery mode"
+       files:
+       - path: /run/cos/recovery_mode
+         content: "1"
+         permissions: 0600
+         owner: 0
+         group: 0
+   boot:
+     - name: "Recovery"
+       if: '[ -f "/run/cos/recovery_mode" ]'
+       hostname: "cos-recovery"
+       commands:
+       - |
+            source /etc/os-release
+            echo >> /etc/issue
+            echo "You are booting from recovery mode. Run 'cos-reset' to reset the system to $VERSION" >> /etc/issue
```

Comment:
does this version need to be evaluated? or is just as an example?

Reply:
this is not really part of this change, as I have just copied the files from https://github.com/kairos-io/packages/tree/main/packages/system/cloud-config/oem here, however it needs to be evaluated (that's why it sources /os-release)